monkey/infection_monkey/monkey.py

```diff
@@ -212,6 +213,7 @@ def _build_puppet(self) -> IPuppet:
         puppet.load_plugin("mssql", MSSQLFingerprinter(), PluginType.FINGERPRINTER)
         puppet.load_plugin("smb", SMBFingerprinter(), PluginType.FINGERPRINTER)
         puppet.load_plugin("ssh", SSHFingerprinter(), PluginType.FINGERPRINTER)
+        puppet.load_plugin("SSHExploiter", SSHExploiter, PluginType.EXPLOITER)
```

Comment:
This should be constructed here. The plugin can't be loaded into the registry otherwise.